src/store/blog/state.js

```diff
@@ -1,5 +1,95 @@
 export default {
   coverImageURL: null,
   pageOfBlogPosts: [],
-  numberOfPages: 0
+  numberOfPages: 0,
```

Comment:
This is dummy data we are using for now. Will eventually be getting this information from the backend.

We should also include a `slug` so that we have a show page to link to in the future.